### PR TITLE
Updated linguist for better representation of source language

### PR DESCRIPTION
**Pull request by paniash**
This PR updates GitHub linguist to showcase python to be the source language of the package. Currently, it provides a false impression of jupyter notebooks to constitute the majority of the package code, which isn't the case.

### CODE REVIEW

File: .gitattributes [hunk 1]
```diff
@@ -1,9 +1 @@
-* text=auto
-
-*.py text
-*.ipynb text
-*.txt text
-*.md text
-
-*.png binary
-*.jpg binary
```
Comment (paniash) [inline .gitattributes:9]: These lines do not affect linguist and can be removed.

Comment (Phionx) [reply .gitattributes:9]: So, this file is necessary to handle line endings across different Operating Systems.
Read more at: https://docs.github.com/en/get-started/getting-started-with-git/configuring-git-to-handle-line-endings

Comment (paniash) [reply .gitattributes]: Ah, I didn't know that. Thanks!

File: .gitattributes [hunk 2]
```diff
@@ -7,3 +7,5 @@
 
 *.png binary
 *.jpg binary
+
+*.ipynb linguist-vendored
```
Comment (Phionx) [inline .gitattributes:11]: Hm, I think the `linguist-vendored` attribute is correct [1], but you may have to include it on the same line as `*.ipynb text`. Currently, it doesn't seem that the Jupyter Notebooks are being excluded from linguistics, as shown in the attached screenshot of your fork.

[1] https://stackoverflow.com/questions/40659265/using-gitattributes-for-linguist-examples

Screen Shot 2021-08-30 at 1 20 28 AM

**Review by Phionx — CHANGES_REQUESTED** [review]

**Pull request closed without merging**